courses/models.py

```diff
@@ -55,35 +56,37 @@ def with_text_id(self, text_id):
         """Applies a filter for the Program's readable_id"""
         return self.filter(readable_id=text_id)
 
-
-class ProgramManager(models.Manager):  # pylint: disable=missing-docstring
-    def get_queryset(self):
-        """Manager queryset"""
-        return ProgramQuerySet(self.model, using=self._db)
-
-    def live(self):
-        """Returns a queryset of Programs with live=True"""
-        return self.get_queryset().live()
-
-    def with_text_id(self, text_id):
-        """Returns a queryset filtered by text id (i.e.: readable_id)"""
-        return self.get_queryset().with_text_id(text_id)
+    def for_product_serializer(self):
+        """Return a QuerySet that is configured for serialization via ProductSerializer"""
+        return self
 
 
 class CourseQuerySet(models.QuerySet):  # pylint: disable=missing-docstring
     def live(self):
         """Applies a filter for Courses with live=True"""
         return self.filter(live=True)
 
+    def prefetch_unexpired_runs(self):
+        """Prefetch unexpired runs"""
+        return self.prefetch_related(
+            models.Prefetch(
+                "courseruns",
+                queryset=CourseRun.objects.unexpired(),
```

Comment:
When I run `CourseRun.objects.unexpired()` in the shell, I see this error:

    In [2]: CourseRun.objects.unexpired()                                           
    ---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-2-0e719d556b3c> in <module>
    ----> 1 CourseRun.objects.unexpired()
    
    AttributeError: 'ManagerFromCourseRunQuerySet' object has no attribute 'unexpired'